documentation/mkdocs.yml

```diff
@@ -4,7 +4,10 @@ nav:
     - 'Getting Started With Project "Piper"' : guidedtour.md
     - Configuration: configuration.md
     - 'Pipelines':
-        - 'General purpose pipeline':
+        - 'ABAP Environment pipeline':
+          - 'Introduction': pipelines/abapEnvironment/introduction.md
+          - 'Configuration': pipelines/abapEnvironment/configuration.md
+          - 'General purpose pipeline':
```

Comment:
Are you sure that intendation is correct here? If i take a raw look at the file I'm not sure it is

Reply:
I think `General Purpose Pipeline` needs to have 1 less intendation if im not mistaking

Reply:
👍 Changed it